usaspending_api/common/filters/time_period/time_period.py

```diff
@@ -1,28 +1,29 @@
 from abc import ABC, abstractmethod
+from types import List
 
 
-class ITimePeriod(ABC):
+class TimePeriod(ABC):
     """Time period interface for storing data related to a time period filter."""
 
     def __init__(self, *args, **kwargs):
```

Comment:
Per the comment about using dates, is there any way to strongly type the input parameters? It diminishes the usefulness of this if impl can go in many directions and isn't required to comply with some basics like a start_date and end_date in the constructor.